pkg/scheduling/volumeusage.go

```diff
@@ -135,7 +132,7 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, volumeName str
 	// In either of these cases, a PV must have been previously bound to the PVC and has since been removed. We can
 	// ignore this PVC while computing limits and continue.
 	if storageClassName == "" {
-		logging.FromContext(ctx).Errorf("failed updating volume limits for volume with unbound PVC, no storage class specified")
+		logging.FromContext(ctx).With("pod", pod.Name).With("pvc", pvc.Name).Errorf("failed updating volume limits for volume with unbound PVC, no storage class specified")
```

Comment:
```suggestion
		logging.FromContext(ctx).With("pod", pod.Name, "pvc", pvc.Name).Errorf("failed updating volume limits for volume with unbound PVC, no storage class specified")
```

Prefer this over calling `With()` twice